pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -113,6 +115,14 @@
                       'supported. '
                       'Default: No properties modified.')]
 
+keybinding_key_option = [              # pylint: disable=invalid-name
+    click.option('-k', '--key', type=str, metavar='KEYNAME=VALUE',
+                 required=False, multiple=True,
+                 help='Value for a key in keybinding of CIMInstanceName. '
```

Comment:
I suggest not to talk about pywbem CIM object classes (CIMInstanceName", but to say "CIM instance name" instead.

Reply:
DONE